mage_ai/data_preparation/models/pipeline.py

```diff
@@ -133,45 +134,55 @@ def create(self, name, pipeline_type=PipelineType.PYTHON, repo_path=None):
         return pipeline
 
     @classmethod
-    def duplicate(cls, source_pipeline: 'Pipeline', duplicate_pipeline_name: str):
-        duplicate_pipeline = cls.create(
-            duplicate_pipeline_name,
+    def duplicate(
+        self,
```

Comment:
classmethods are generally supposed to use `cls` instead of `self`

Reply:
So i saw the python linting suggest that also, but I changed it to "self" to make it consistent with our other methods. I'll change it back to `cls` though.